src/shared/components/Button/Button.style.ts

```diff
@@ -127,12 +197,26 @@ export const StyledIcon = styled(Icon)<IconStyleProps>`
     margin-left: 10px;
   }
   filter: ${(props) => (props.disabled ? 'brightness(0.7)' : null)};
+  ${iconSizeFromProps}
 `
 export const StyledButton = styled.button<ButtonStyleProps>`
+  --size-without-text-small: 32px;
```

Comment:
I don't think we necessarily need to put those in CSS vars. I meant storing them in regular JS vars. I don't think those values will be used outside of this component, so we don't need to worry about sharing